examples/src/main/java/ai/djl/examples/inference/face/model/Landmark.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+ * with the License. A copy of the License is located at
+ *
+ * http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+ * OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+
+package ai.djl.examples.inference.face.model;
+
+import ai.djl.modality.cv.output.Point;
+import java.util.List;
+
+public class Landmark {
```

Comment:
You can place this class under https://github.com/awslabs/djl/tree/master/api/src/main/java/ai/djl/modality/cv/output.  You may also need to follow other classes in the same directory to have annotation on on public classes and methods